### DISCUSSION TITLE
Wordpress plugin - More verses

### DISCUSSION THREAD

**Discussion by bolach** [Bibleup/bibleup#50] [Ideas]
Hi,
Thanks for the wordpress plugin! 
If we try to cite several verses, the plugin shows an incomplete link.
Luke 1:13 -> Works
Luke 1:13-17 - Doesn't Work

Do you think it is possible to include everything (all text from 13-17) in the popup window, in future releases?

Example:
https://i.imgur.com/UMo97L0.jpeg

**Comment by Bukunmikuti** [comment]
Hi @bolach
BibleUp works well on those examples and much more. The demo page confirms that ([bibleup.netlify.app/demo](https://bibleup.netlify.app/demo)).
I checked and it works on my test blog also 
<img src="https://github.com/bibleup/bibleup/assets/67844971/949d6a77-1aee-4268-a97b-dc64795c5e93" height="300px" width="300px" />


Make sure there is no space around the dash
Luke 1:13 - 17 ❌
Luke 1:13- 17 or Luke 1:13-17 ✅ (only one space before or after dash or no space at all) 
You may deactivate and reactivate the plugin or check if it works with other references. Kindly let me know if it resolves.

**Reply by bolach** [reply]
It was not working. I disabled like you told, activated and now works :)
Thank you

**Reply by Bukunmikuti** [reply]
Glad it works now. I am working on more Bible versions for the next release. 
One request please, would you kindly drop an honest review for the plugin whenever you have the chance. It would make more Christian publishers find BibleUp 🙏
